changelogs/8.12.asciidoc

```diff
@@ -13,6 +13,7 @@ https://github.com/elastic/apm-server/compare/8.11\...8.12[View commits]
 ==== Breaking Changes
 - The unsupported apm_data_stream_migration ingest pipeline has been removed {pull}12102[12102].
 - "publish_ready" is always false in the "GET /" response until events are received by apm-server {pull}12150[12150]
+- (Known issue) `traces-apm@custom` ingest pipeline is now applied to data streams `traces-apm.sampled-*` and `traces-apm.rum.*` and applied twice for `traces-apm-*`. See Fleet changelog for details.
```

Comment:
Does it clearly convey that it is a bug? Feel free to modify the wording.

Reply:
How about something like: "A bug has been introduced, so that `traces-apm@custom` ingest pipeline is now additionally applied to data streams `traces-apm.sampled-*` and `traces-apm.rum.*` and applied twice for `traces-apm-*`. See more details and information around a fix in the Known issue section" 

And then add it to the Known issue section, and once `8.12.1` is released, we update the Known issue section with a note about the fix.

Reply:
Updated. Known issues lives in https://github.com/elastic/observability-docs/blob/main/docs/en/observability/apm/known-issues.asciidoc .